.github/workflows/example-testing.yml

```diff
@@ -1,54 +0,0 @@
-# Notes:
```

Comment:
These tests have been moved to `isofit/test/test_examples.py`, labeled with a `pytest` marker, and are executed as part of the `pip` package test workflow. They can be selected with:

```console
$ pytest -m examples
```